awscli/examples/application-signals/untag-resource.rst

```diff
@@ -0,0 +1,13 @@
+**To remove one or more tags from the specified resource**
+
+The following ``untag-resource`` example removes one or more tags from the specified resource. ::
+
+    aws application-signals untag-resource \
+        -—resource-arn "arn:aws:application-signals:us-east-1:123456789101:slo/SLOName" \
+        -—tag-keys "test"
```

Comment:
Both of these ended up with a non-ASCII dash, can you make sure they're only `--`?